src/helpers.js

```diff
@@ -1,13 +1,13 @@
 const playwright = require("playwright");
-const { devices } = require("@replayio/playwright");
 const fs = require("fs");
 require("dotenv").config();
 
-let browserName = process.env.PLAYWRIGHT_CHROMIUM ? "chromium" : "firefox";
+process.env.RECORD_ALL_CONTENT = 1;
+
+let browserName = "chromium";
 const launchOptions = {
   headless: !!process.env.PLAYWRIGHT_HEADLESS,
-  ...devices[browserName === "firefox" ? "Replay Firefox" : "Replay Chromium"]
-    .launchOptions,
+  executablePath: process.env.HOME + "/tmp/Replay-Chromium.app/Contents/MacOS/Chromium",
```

Comment:
Using `RECORD_REPLAY_DIRECTORY` and `REPLAY_CHROMIUM_EXECUTABLE_PATH` should be sufficient. If it is not, we should fix that.